client/js/components/data_page.jsx

```diff
@@ -1,14 +1,17 @@
 var React = require("react");
 var _ = require("lodash");
 var reactBootStrap = require("react-bootstrap");
+var AppStore = require("../store/app_store.js");
 
 var UploadButton = React.createClass({
-  fileHandler: function(event) {
-    console.log(this.files);
-  },
   componentDidMount: function() {
+    var schemaName = this.props.schemaName;
+    function fileHandler(event){
```

Comment:
`event` is unused, consider removing. the missing space in `){` violates our style guide.